src/include/utime.h

```diff
@@ -63,6 +63,10 @@ class utime_t {
     }
   }
 
+  static inline constexpr utime_t max() {
+    return utime_t{time_t{std::numeric_limits<uint32_t>::max()}, 0};
```

Comment:
max() should really be >= to every utime_t instance, set the nsec member to max as well.  This could be its own commit.

Reply:
Done.

Reply:
https://github.com/ceph/ceph/pull/53038